action/protocol/managers.go

```diff
@@ -76,16 +76,26 @@ type (
 		States(...StateOption) (uint64, state.Iterator, error)
 		ReadView(string) (interface{}, error)
 	}
+	
+	// StateWriter defines an interface to write stateDB
+	StateWriter interface {
```

Comment:
it is better to have
```
StateWriter interface {
    StateReader
    PutState...
    DelState...
}
```

but I don't see the value distinguishing the statewriter and statemanager.

Reply:
sure,I can change it to this